sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_aws_sdk.ipynb

```diff
@@ -0,0 +1,889 @@
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "processing_job_name = create_processing_job(bias_analysis_config_path)\n",
+    "\n",
+    "# Wait for processing job to complete\n",
+    "status = sagemaker_client.describe_processing_job(ProcessingJobName=processing_job_name)\n",
+    "\n",
+    "while status[\"ProcessingJobStatus\"] == \"InProgress\":\n",
+    "    status = sagemaker_client.describe_processing_job(ProcessingJobName=processing_job_name)\n",
+    "    print(status[\"ProcessingJobStatus\"])\n",
+    "    time.sleep(60)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Viewing the Bias Report\n",
+    "In Studio, you can view the results under the experiments tab.\n",
+    "\n",
+    "<img src=\"./recordings/bias_report.gif\">\n",
+    "\n",
+    "Each bias metric has detailed explanations with examples that you can explore.\n",
+    "\n",
+    "<img src=\"./recordings/bias_detail.gif\">\n",
+    "\n",
+    "You could also summarize the results in a handy table!\n",
+    "\n",
+    "<img src=\"./recordings/bias_report_chart.gif\">\n",
+    "\n",
+    "If you're not a Studio user yet you can access the complete analysis report at the following S3 bucket.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "analysis_result_path"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Explaining Predictions\n",
+    "There are expanding business needs and legislative regulations that require explanations of _why_ a model made the decision it did. SageMaker Clarify uses SHAP to explain the contribution that each input feature makes to the final decision."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Configure a SageMaker Clarify Processing Job Container's Input and Output Parameters "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "explainability_analysis_config_path = \"s3://{}/{}/explainability_analysis_config.json\".format(\n",
+    "    bucket, prefix\n",
+    ")\n",
+    "analysis_result_path = \"s3://{}/{}/output\".format(bucket, prefix)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Configure Analysis Config\n",
+    "For our example use case we will be using the following analysis config:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!echo\n",
+    "!cat explainability_analysis_config.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`explainability_analysis_config.json` here contains configuration values for detecting bias using a clarify container:\n",
```

Comment:
Major comment: "for detecting bias using a clarify container" => "for computing feature attribution using a SageMaker Clarify job".